source/javascripts/controllers/_StackController.js.erb

```diff
@@ -26,6 +26,7 @@
 				"gen1": "<%= data[:strings][:stack][:machine_type_notes][:gen1] %>",
 				"gen2": "<%= data[:strings][:stack][:machine_type_notes][:gen2] %>"
 			};
+			viewModel.betaMachineTypeIds = ['g2-m1.8core', 'g2-m1-max.5core'];
```

Comment:
The beta warning in question is specific to Apple Silicon. I'd rename a few things here -- `appleSiliconMachineTypeIds`, `isAppleSiliconMachineType`, `apple_silicon_beta_warning`, etc.

Reply:
@jakerobb updated